relays/bin-substrate/src/cli/init_bridge.rs

```diff
@@ -244,9 +244,14 @@ impl BridgeInitializer for PolkadotToPolkadotBulletinCliBridge {
 	fn encode_init_bridge(
 		init_data: <Self::Engine as Engine<Self::Source>>::InitializationData,
 	) -> <Self::Target as Chain>::Call {
-		relay_polkadot_bulletin_client::RuntimeCall::BridgePolkadotGrandpa(
-			relay_polkadot_bulletin_client::BridgePolkadotGrandpaCall::initialize { init_data },
-		)
+		type RuntimeCall = relay_polkadot_bulletin_client::RuntimeCall;
```

Comment:
And yet another change here: bulletin chain has the sudo pallet - let's use it to initialize bridge.